src/Sulu/Bundle/ContentBundle/Resources/public/js/components/single-internal-link/main.js

```diff
@@ -185,46 +184,27 @@ define([], function() {
         /**
          * initialize column navigation
          */
-        overlayOkCallback = function () {
-            this.sandbox.emit('husky.column-navigation.' + this.options.instanceName + '.get-marked', function (markedCollections) {
-                if (Object.keys(markedCollections).length > 0) {
-                    setData.call(this, Object.keys(markedCollections)[0]);
-                    loadSelectedNode.call(this);
-                    this.sandbox.emit(DATA_CHANGED.call(this), this.data, this.$el);
-
-                    this.sandbox.emit('husky.overlay.single-internal-link.' + this.options.instanceName + '.close');
-                } else {
-                    setData.call(this, '');
-                    this.$input.val('');
-
-                    this.sandbox.emit(DATA_CHANGED.call(this), this.data, this.$el);
-                }
-            }.bind(this));
-        },
-
-        /**
-         * initialize column navigation
-         */
         initColumnNavigation = function() {
-            var url = getSelectUrl(this.options.columnNavigationUrl, this.data);
-
             this.sandbox.start(
                 [
                     {
-                        name: 'column-navigation@husky',
+                        name: 'content-datasource@sulucontent',
                         options: {
                             el: getId.call(this, 'columnNavigation'),
                             selected: this.data,
-                            url: url,
+                            selectedUrl: this.options.selectedUrl,
+                            rootUrl: this.options.rootUrl,
+                            resultKey: 'nodes',
```

Comment:
I mean previously there was a line like `resultKey: this.options.resultKey`, and now you are hardcoding the value.